src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -422,7 +412,6 @@ function ComposerWithSuggestions(
                         suggestionsRef.current.resetSuggestions();
                     }
                     insertedEmojisRef.current = [...insertedEmojisRef.current, ...newEmojis];
```

Comment:
We are not using insertedEmojisRef.current anywhere, so we can remove it from the file.